### DISCUSSION TITLE
Can't get GH Pages to serve up /.well-known/apple-app-site-association

### DISCUSSION THREAD

**Discussion by JetForMe** [community/community#45034] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I've set up a super basic GH pages site (the second one I've done). It's serving up regular content correctly, but won't serve up files in `/.well-known/apple-app-site-association`. I found some discussion online that said I had to include a `_config.yml` file with

```
include: [".well-known"]
```

but that doesn’t seem to have helped. Any suggestions?

**Comment by jge162** [comment]
@JetForMe It sounds like you may have missed a step in the setup process for serving up files in the ".well-known" directory. In addition to including the "include" line in your _config.yml file, you also need to make sure that your ".well-known" directory is at the root level of your GitHub Pages repository and that the files within it are accessible at the root level of your GitHub Pages website.
Another thing to check is the permissions of your file and folder. Make sure they are not blocked or private.
If that doesn't work, you can try to clear your browser cache and hard refresh the page.

**Reply by JetForMe** [reply]
To be clear, should `.well-known` be in the repo root, or in `/docs` (it's currently in the latter):

```
repo/ (main) $ ls -R
README.md    _config.yml  docs/

./docs:
CNAME       index.html     .well-known/

docs/.well-known:
apple-app-site-association
```

**Reply by JetForMe** [reply]
And fwiw, moving `.well-known` to the root of my repo didn't seem to change anything.

**Reply by PythonCoderAS** [reply]
Try putting an `index.html` file inside of `.well-known` and putting it inside `docs`.

**Comment by JetForMe** [comment, marked as answer]
After further experimentation, it seems `_config.yml` needs to live inside the deployment source directory (e.g. `/docs`).

**Comment by mikolaj1982** [comment]
is this your final structure also trying to make apple-app-site-association work with github pages, thanks

```
repo/ (main) $ ls -R
README.md    _config.yml  docs/

./docs:
CNAME       index.html     .well-known/

docs/.well-known:
apple-app-site-association
```

**Reply by JetForMe** [reply]
I've since stopped using GH pages for that purpose, but I think that's right.
